package.json

```diff
@@ -1,33 +1,17 @@
 {
-  "ava": {
-    "failFast": false,
-    "files": [
-      "dist/tests/**/*.js"
-    ],
-    "timeout": "1m"
-  },
-  "dependencies": {
-    "@azure/functions": "^1.0.3",
-    "@destinationstransfers/ntp": "^2.0.0",
-    "@octokit/rest": "^16.28.7",
```

Comment:
We don't need any of these as dependencies?

Reply:
not in the root. remember that this is now a monorepo, each package has its own package.json